.swiftlint.yml

```diff
@@ -101,9 +99,13 @@ file_name:
 #     - extension
 
 identifier_name:
+  max_length: 60
   excluded:
     - id
 
+nesting:
```

Comment:
Please move this under `line_length` to keep the rule configurations sorted alphabetically.

Reply:
@Dschee fixed it. ;)